src/software/ai/navigator/obstacle/robot_navigation_obstacle_factory.cpp

```diff
@@ -160,9 +160,11 @@ RobotNavigationObstacleFactory::createDynamicObstaclesFromMotionConstraint(
         case TbotsProto::MotionConstraint::AVOID_BALL_PLACEMENT_INTERFERENCE:;
             if (world.gameState().getBallPlacementPoint().has_value())
             {
-                obstacles.push_back(createFromBallPlacement(
+                auto obstacles_list = createFromBallPlacement(
```

Comment:
just curious: was this the change needed that fixed ball placement - we're there too many obstacles?